pymica_tests/importers_pymica_test.py

```diff
@@ -0,0 +1,44 @@
+'''Tests for importers.py
+'''
+import unittest
+
+
+import dateutil.parser
+
+from pymica.io.importers import \
+    import_api_smc  # pylint: disable=E0611
+
+
+class TestImporters(unittest.TestCase):
+
+    url = 'http://smcawrest01:8080/ApiRestInterna/xema/v1/mesurades/' \
+          'dades/variables'
+    url_meta = "http://smcawrest01:8080/ApiRestInterna/xema/v1/mesurades/metadades/estacions"
+
+    station_list = ["C6","C7","C8","CE","CW","D1","D7","D8","D9","DI","DK","DQ","H1","MR","U3","UH","UJ","UM","V1","V8",
+                "VA","VB","VC","VD","VM","VY","VQ","W4","W8","WB","WC","WD","WJ","WL","WP","WV","WZ","XA","XB","XD",
+                "X1","XE","XI","XM","XP","XR","XX","Y6","YD","YJ","YF","YH","YE","YL","YO"]
+    
+    #station_list = ["ZD"]
+    station_list = None
+    
+    variable = '2t'
+
+    date = "20210401T0000Z"
+    date = dateutil.parser.parse(date)
+
```

Comment:
This test only checks station_list = None. There should be two tests, one for None and one for two or three stations. station_list in class is not needed.